content/docs/04-clusters/03-edge/04-edgeforge-workflow/12-build-artifacts.md

```diff
@@ -0,0 +1,309 @@
+
+  For example, the screenshot below highlights the cluster profile and project IDs in a cluster details page URL. 
+
+  ![A screenshot highlighting the cluster profile and project IDs in the URL of the cluster details page.](/clusters_edge_edge-forge-workflow_build-images_build-artifacts_url.png)
+
+
+
+16. Set the profile ID using the following command. Replace the `[USE-YOUR-PROFILE-ID_HERE]` placeholder with your cluster profile ID. This command sets the profile ID, which identifies the specific cluster profile that you want to use for creating the offline content bundle. 
+<br />
+
+  ```bash
+  export PROFILE_ID=[USE-YOUR-PROFILE-ID_HERE]
+  ```
+
+
+17. Set the project ID by issuing the following command. Replace the `[USE-YOUR-PROJECT-ID_HERE]` placeholder with your cluster profile ID.
+The offline content bundle will be associated with the project ID you will use in the following command. 
+<br />
+
+  ```bash
+  export PROJECT_ID=[USE-YOUR-PROJECT-ID_HERE]
+  ```
+
+
+18. Build the offline content bundle using the following command. The content bundle will have the configuration and settings for the Edge cluster, including the operating system image, Kubernetes distribution, any additional packs, and manifest files specified in your cluster profile you wish to deploy to the Edge cluster.
+
+  The command below uses the palette-edge tool to build the offline content bundle. The `--api-key`, `--project-id`, and `--cluster-profile-ids` options specify the necessary parameters for the build process. The `--palette-endpoint` option specifies the endpoint URL for the Spectro Cloud API. The `--outfile` option specifies the name of the output file for the content bundle.
+<br />
+
+  ```bash
+  palette-edge build --api-key $API_KEY \
+  --project-id $PROJECT_ID \
+  --cluster-profile-ids $PROFILE_ID \
+  --palette-endpoint api.spectrocloud.com \
+  --outfile content
+  ```
+
+
+19. Use the command below to list all files in the current directory, including the newly built content bundle. The content bundle will have the following naming convention, `content-[randon-string]`, for example, **content-8e61a9e5**. The command below will verify that you successfully created the content bundle with the specified name.
+<br />
+
+  ```bash
+  ls -al
+  ```
+
+
+20. List the details of the content bundle using the following command. Replace the `content-[randon-string]` with the content bundle in your current directory. The command below will provide information such as file size, permissions, and modification date.
+<br />
+
+  ```bash
+  ls -l content-[randon-string]
+  ```
+
+
+21. Issue the following command to execute the **earthly.sh** script with elevated privileges. The `+iso` option specifies the build target. This command will generate an ISO image from the content bundle and other configurations you have specified in the **.arg** and **user-data** files. 
```

Comment:
Should we mention that it takes a while 😅 . You used this sentence in the Edge artifacts guide:
_This command may take up to 15-20 minutes to finish depending on the resources of the host machine_.